user-guide/Advanced_Modules/DOM/DomHelper_class.md

```diff
@@ -88,6 +88,95 @@ while (pagingHelper.MoveToNextPage())
 
+Once you have read the `DomInstance`, you can get or alter the field values. See the [examples](xref:DOM_Altering_values_of_a_DomInstance) for more info on how to do this.
+
+> [!TIP]
+> When you need to read multiple `DomInstances` using their IDs, it is more efficient to build one big OR filter and read them in one call instead of retrieving them one by one. There is a helper method available for this called `Tools.RetrieveBigOrFilter()` see the [DOM best practices page](xref:DOM_best_practices#try-to-limit-the-number-of-crud-calls) for more info.
+
+#### Filtering
+
+To specify which `DomInstances` need to be retrieved from the database, you can build filters using the `FilterElement` structures. To start a filter, you need to specify the field you want to filter on using the `DomInstanceExposers` class. Once a field is selected, you can choose the comparer and provide a value. These filters can then be concatenated to each other with AND and OR conditions.
+
+**Base properties:**
+
+|Field                |Type        |Supported Comparer |
+|---------------------|------------|-------------------|
+|Id                   |string      |Equal, NotEqual |
+|SectionDefinitionIds |List\<Guid> |Contains, NotContains |
+|SectionIds           |List\<Guid> |Contains, NotContains |
+|DomDefinitionId      |Guid        |Equal, NotEqual |
+|Name                 |string      |Equal, NotEqual, Contains, NotContains |
+|StatusId             |string      |Equal, NotEqual, Contains, NotContains |
+|LastModified         |DateTime    |LessThan, GreaterThan |
+|LastModifiedBy       |string      |Equal, NotEqual, Contains, NotContains |
+|CreatedAt            |DateTime    |LessThan, GreaterThan |
+|CreatedBy            |string      |Equal, NotEqual, Contains, NotContains |
+
+**Fields:**
+
+These are the supported comparers when filtering on `DomInstance` values using the `DomInstanceExposers.FieldValues.DomInstanceField()` exposer.
+
+|Field Type                |Supported Comparer |
+|--------------------------|-------------------|
+|string                    |Equal, NotEqual, Contains, NotContains |
+|double                    |Equal, NotEqual, LessThan, LessThanOrEqual, GreaterThan, GreaterThanOrEqual |
+|long                      |Equal, NotEqual, LessThan, LessThanOrEqual, GreaterThan, GreaterThanOrEqual |
+|TimeSpan                  |Equal, LessThan*, GreaterThan* |
+|Boolean                   |Equal |
+|DateTime                  |Equal, NotEqual, LessThan, GreaterThan |
+|GenericEnum (int)         |Equal, NotEqual, LessThan, LessThanOrEqual, GreaterThan, GreaterThanOrEqual |
+|List GenericEnum (int)    |Contains |
+|GenericEnum (string)      |Equal, NotEqual, Contains, NotContains |
+|List GenericEnum (string) |Contains |
+|Guid                      |Equals, NotEquals |
+|List Guid                 |Contains |
+|List string               |Contains |
+
+*Only supported on [STaaS](xref:STaaS)
+
+> [!NOTE]
+> The filters on `string` values are case insensitive.
```

Comment:
Only from 10.4.5, 10.4.0[CU2]. Maybe also mention that there is an option to make the filter case-sensitive? [String fileds case-insensitive RN](https://intranet.skyline.be/DataMiner/Lists/Release%20Notes/DispForm2.aspx?ID=38950)

Reply:
Expanded the note. I would however wait with explaining the case-sensitive feature until STaaS supports it as well.